bokeh/util/logconfig.py

```diff
@@ -18,9 +18,36 @@
 from __future__ import absolute_import
 
 import logging
+import sys
 
 from ..settings import settings
 
 level = settings.py_log_level()
+bokeh_logger = logging.getLogger('bokeh')
+root_logger = logging.getLogger()
+
 if level is not None:
-    logging.basicConfig(level=level)
+    bokeh_logger.setLevel(level)
+
+if not (root_logger.handlers or bokeh_logger.handlers):
+    # No handlers configured => at least add a printer to sys.stderr for
```

Comment:
Ah, I was not aware. Thanks, I will test this PR locally to confirm validation messages are emitted. Can you merge/rebase on master so the fix for the upstream CI issues can be brought in.